test/test_main.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, framework: ops.Framework):
                     ops.main(MyCharm)
 
         assert mock_charm_event.call_count == 1
-        return mock_charm_event.call_args[0][0]
+        return mock_charm_event.call_args[0][1]
```

Comment:
Can you explain this change?

Reply:
The signature of `_emit_charm_event` in `ops._main` has changed from: `_emit_charm_event(self, event_name: str)` to `_emit_charm_event(self, charm: _charm.CharmBase, event_name: str)`. That means that the name is now `[1]` rather than `[0]`.

I could switch the arguments around so that `event_name` is still first, but it seemed more consistent to have the charm first, and also more logical in that the name is "emit charm event" so "charm then event" as the argument order aligns.

Reply:
thanks